changelog.d/20231213_163231_faraz.maqsood_faraz_add_instructions_related_to_programs.md

```diff
@@ -0,0 +1 @@
+- [Documentation] Add instrcutions in the readme related to Programs tab on LMS. (by @Faraz32123)
```

Comment:
Are these changelogs needed? I do not see any other doc in the directory.
nit: instructions

Reply:
@regisb can tell better, as what i did is a change in readme(its a documentation).

Reply:
Yeah, we don't mention doc updates in the changelog. That's because the goal of the changelog is _not_ to track every change in the repo. The changelog is designed for users who upgrade from one release to the next and need to know what's new in every release.

Reply:
best.